### PR TITLE
Fixup Google Analytics DocumentationURLs and Spec Names

### PR DESCRIPTION
**Pull request by evantahler**
[Slack Thread](https://airbytehq-team.slack.com/archives/C032Y32T065/p1678837968389009?thread_ts=1678824228.772719&cid=C032Y32T065)

### CODE REVIEW

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml [hunk 1]
```diff
@@ -760,7 +760,7 @@
   sourceDefinitionId: eff3616a-f9c3-11eb-9a03-0242ac130003
   dockerRepository: airbyte/source-google-analytics-v4
   dockerImageTag: 0.1.34
-  documentationUrl: https://docs.airbyte.com/integrations/sources/google-analytics-universal-analytics
+  documentationUrl: https://docs.airbyte.com/integrations/sources/google-analytics-data-api
```
Comment (erohmensing) [inline airbyte-config/init/src/main/resources/seed/source_definitions.yaml:763]: Shouldn't this match the docker repository? This explicitly matches the other docker repository/source module

Comment (erohmensing) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:763]: see 
https://github.com/airbytehq/airbyte/blob/5512befeb1ac2c1d3348dd20df0ed31b11e1c4d3/docs/integrations/sources/google-analytics-data-api.md

vs 

https://github.com/airbytehq/airbyte/blob/5512befeb1ac2c1d3348dd20df0ed31b11e1c4d3/docs/integrations/sources/google-analytics-v4.md

google-analytics-v4.md contains the docs for universal analytics/source-google-analytics-v4

**Review by erohmensing — CHANGES_REQUESTED** [review]

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml (hunk 1 above)
Comment (erohmensing) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:763]: This also follows what Alex said: 
> Changes needed:
Google Analytics (UA)
connector definition id: eff3616a-f9c3-11eb-9a03-0242ac130003
documentation url: https://docs.airbyte.com/integrations/sources/google-analytics-v4/
Google Analytics 4
connector definition id: 3cc2eafd-84aa-4dca-93af-322d9dfeec1a 
documentation url: https://docs.airbyte.com/integrations/sources/google-analytics-data-api/

Comment (erohmensing) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:763]: + URL on line 771 should be `https://docs.airbyte.com/integrations/sources/google-analytics-data-api/`

Comment (evantahler) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:763]: OH! Thank you, I read that wrong.
... so the v4 connector doesn't get the "V4" docs.  Got it :D

Comment (erohmensing) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml]: The GA4 connector doesn't, yeah - the v4 one is universal analytics... for some cursed reason 😅

**Review by erohmensing — APPROVED** [review]
I hope this finally resolves the confusion 🤞

**Comment by evantahler** [comment]
> ERROR:root:The following ['generally_available', 'beta'] connectors must include allowedHosts: [source-google-analytics-v4]

I'm ignoring this pre-existing problem and merging this documentation change

**Pull request merged**